Cannot install, missing package.json

## What is the problem

Cannot install using `npm install`
```js
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path [...]/Epitech-MSc_Competencies-Overview/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '[...]/Epitech-MSc_Competencies-Overview/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

Probably because of missing package.json
I tried `npm ci` instead, to no avail.

## How to reproduce
After git cloning the repo, try to `npm install` from the repo.

## What is expected
Using npm install, the packages defined in package.json (and package-lock.json) are installed, and the software can be used.

--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@
 
 ## Setup
 * You will need [NodeJS](https://nodejs.org/en/download/) to be installed 
-* Run `npm install -g typescript` and `npm install` in this directory
+* Run `npm install` in this directory
 * Get your competency tree and store it in the `compentency_tree.html` file
   * Go to Moodle
   * Go to the competency page
@@ -15,7 +15,7 @@
 
 ## Start
 
-Run `tsc ./competency_parser.ts` and then `node ./competency_parser.js`
+Run `npm start`
 
 ## Output examples
 

--- a/package.json
+++ b/package.json
@@ -1,0 +1,34 @@
+{
+  "name": "competency_overview",
+  "version": "1.0.0",
+  "description": "Parses the competency tree from Moodle to show, filter and order competencies and skills",
+  "main": "competency_parser.js",
+  "dependencies": {
+    "@types/node": "^16.4.10",
+    "@types/prompt": "^1.1.1",
+    "node-html-parser": "^4.1.2",
+    "prompt": "^1.1.0",
+    "source-map-loader": "^3.0.0",
+    "ts-loader": "^9.2.4",
+    "typescript": "^4.3.5"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "start": "tsc ./competency_parser.ts && node ./competency_parser.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/HaberkornJonas/Epitech-MSc_Competencies-Overview.git"
+  },
+  "keywords": [
+    "Epitech",
+    "MSc-Pro",
+    "Competencies"
+  ],
+  "author": "Jonas Haberkorn",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/HaberkornJonas/Epitech-MSc_Competencies-Overview/issues"
+  },
+  "homepage": "https://github.com/HaberkornJonas/Epitech-MSc_Competencies-Overview#readme"
+}

